### DISCUSSION TITLE
Welcome to filepond-plugin-pdf-preview Discussions!

### DISCUSSION THREAD

**Discussion by Adri-Glez** [Adri-Glez/filepond-plugin-pdf-preview#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Delta-Band** [comment]
Hi, I just installed and used this plugin successfully when uploading a pdf. (React App)
but when trying to pass it back from the server and preview it as a local existing file I get an error that `file` is not defined.
I am trying to create a file instance from the already existing pdf returning as a URL from the server.
I am trying to do something like this:
```
function mockFile(url) {
  const blob = new Blob([url], {
    type: 'application/pdf'
  });
  return {
    source: blob,
    options: {
      type: 'local',
    }
  };
}
// in the render method:
<FilePond
  files={mockFile(url)}
/>
```
any thoughts on how may get this to work?

**Comment by Sogugui** [comment]
Hi.
I found a bug using this plugin: Safari does not support it and freeze the page when tries to load the pdf that was already loaded before. (It failed doing the get to previsualize)
